src/styles/ComponentPage.scss

```diff
@@ -50,5 +45,8 @@
 }
 
 .demo-content {
+  width: 1000px;
+  margin: 0 auto;
+  padding: 24px;
```

Comment:
Make this `0 16px` so that the top padding doesn't overlap and the text lines up with the button

![image](https://user-images.githubusercontent.com/5939574/39722612-360ce81e-51f8-11e8-8c14-2cce1eb30133.png)